pkg/validation/limits.go

```diff
@@ -299,6 +300,9 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	_ = l.MetadataQuerySplitDuration.Set("24h")
 	f.Var(&l.MetadataQuerySplitDuration, "querier.split-metadata-queries-by-interval", "Split metadata queries by a time interval and execute in parallel. The value 0 disables splitting metadata queries by time. This also determines how cache keys are chosen when label/series result caching is enabled.")
 
+	_ = l.IngesterQuerySplitDuration.Set("0s")
+	f.Var(&l.IngesterQuerySplitDuration, "querier.split-ingester-queries-by-interval", "Interval to use for time-based splitting when a request is within the `query_ingesters_within` window; defaults to `split-queries-by-interval` by setting to 0.")
```

Comment:
how do we fallback to `split-queries-by-interval` in case it is set to 0? just curious because I can not find it.

Reply:
https://github.com/grafana/loki/pull/11535/files#diff-5ae98add35d728910d8469070693a406831e482d1597d2a249303fa39d26ec43R120-R123

Effectively we don't handle the splits specially if the limit is 0, which uses `split-queries-by-interval`.